x-pack/test/functional/apps/maps/documents_source/top_hits.js

```diff
@@ -12,13 +12,19 @@ export default function ({ getPageObjects, getService }) {
   const PageObjects = getPageObjects(['maps', 'common']);
   const inspector = getService('inspector');
   const find = getService('find');
+  const security = getService('security');
 
   describe('geo top hits', () => {
     describe('split on string field', () => {
       before(async () => {
+        await security.testUser.setRoles(['global_maps_all', 'test_logstash_reader'], false);
```

Comment:
Same on this test.  Could it pass with `global_maps_read`?

Reply:
@LeeDr there is a configuration change which doesn't work with just read permission for this test. 
```
1)    maps app
   │       
   │         documents source
   │           geo top hits
   │             split on string field
   │               configuration
   │                 "before all" hook for "should update top hits when configation changes":
   │
   │      Error: retry.try timeout: TimeoutError: Waiting for element to be located By(css selector, [data-test-subj="editLayerButton"])
   │ Wait timed out after 10009ms
   │     at /Users/bhavyarajumandya/Desktop/kibana_dev/kibana/node_modules/selenium-webdriver/lib/webdriver.js:842:17
   │     at process._tickCallback (internal/process/next_tick.js:68:7)
   │       at onFailure (/Users/bhavyarajumandya/Desktop/kibana_dev/kibana/test/common/services/retry/retry_for_success.ts:28:9)
   │       at retryForSuccess (/Users/bhavyarajumandya/Desktop/kibana_dev/kibana/test/common/services/retry/retry_for_success.ts:68:13)
   │ 
   │ 
   │2)    maps app
   │       
   │         documents source
   │           geo top hits
   │             split on string field
   │               configuration
   │                 "after all" hook for "should update top hits when configation changes":
   │
   │      Error: retry.try timeout: TimeoutError: Waiting for element to be located By(css selector, [data-test-subj="layerPanelCancelButton"])
   │ Wait timed out after 10044ms
   │     at /Users/bhavyarajumandya/Desktop/kibana_dev/kibana/node_modules/selenium-webdriver/lib/webdriver.js:842:17
   │     at process._tickCallback (internal/process/next_tick.js:68:7)
   │       at onFailure (/Users/bhavyarajumandya/Desktop/kibana_dev/kibana/test/common/services/retry/retry_for_success.ts:28:9)
   │       at retryForSuccess (/Users/bhavyarajumandya/Desktop/kibana_dev/kibana/test/common/services/retry/retry_for_success.ts:68:13)
   │ 
``` 

I need to give it global_maps_all.

Reply:
We found out that we need to keep global_maps_all for the editing of maps layer (logstash) here